### DISCUSSION TITLE
blog/posts/new-beginning

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#252] [文档评论区]
# blog/posts/new-beginning

Zotero 非官方中文维护小组

https://zotero-chinese.com/blog/posts/new-beginning

**Comment by l0o0** [comment]
新征程，新开始🎉

**Comment by xuxuxu-hao** [comment]
已提交您的错误报告。

报告 ID: 280403027

您需要在 Zetero 论坛中发帖，帖子中附上报告 ID、问题描述、重现问题所需的步骤。

未在论坛中提交的错误报告不会被处理。

**Reply by northword** [reply]
请将错误报告至 Zotero 论坛：forums.zotero.org

**Comment by Ada66688** [comment]
请问一下更新插件，更新translator、设置PDFtk路径都尝试了，zotero就是不能抓取文献无论中英文是怎么回事呢

**Comment by alexzorro561** [comment]
真的非常棒！感谢国内有这么一批仁人志士，我很庆幸也很感恩能享受到如此大的便利和福利！我是医学生，不太了解计算机技术，非常感谢！

**Comment by lxcy1234** [comment]
Linter for Zotero
监测到您导入了一个 WebPage 条目，其 URL 中包含了主要学术出版商的域名，\n导入可能存在异常，请确认!      请问这个如何处理？

**Comment by dark729** [comment]
翻译插件不会自动翻译怎么办

**Comment by Junhan9964** [comment]
您好，非常感谢各位的付出。更新完新版本后发现两点以前觉得很方便的功能，现在貌似不见了，所以想提一下。以前的版本在创立独立笔记后，左侧会有目录很方便找到对应的内容。现在没有了，要一直往下翻，不是很方便。另外，以前版本在注释栏里可以直接生成该注释的链接，然后可以放在独立笔记里。现在貌似注释栏里不能直接操作了。虽然复制内容可以直接粘贴到独立笔记，并生成链接。但是外语文献的情况，以前注释栏直接生成的注释链接放在独立笔记里可以现实原文、注释、翻译。 是否方便再次添加，谢谢🙏

**Comment by klxdzh** [comment]
zotero 7.0.20更新为7.0.24版本之后两个程序为什么同时存在呢？

**Comment by Ada66688** [comment]
我也遇到了，无法解决🙂‍↔️



发自我的iPhone


------------------ Original ------------------
From: klxdzh ***@***.***&gt;
Date: Mon,Aug 25,2025 10:00 AM
To: zotero-chinese/wiki ***@***.***&gt;
Cc: Ada66688 ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [zotero-chinese/wiki] blog/posts/new-beginning (Discussion #252)





 
zotero 7.0.20更新为7.0.24版本之后两个程序为什么同时存在呢？

**Reply by klxdzh** [reply]
那控制面板里面卸载7.0.20会影响软件运行么

**Comment by Ada66688** [comment]
会，两个其实是一个，方便问一下你抓取文献功能正常吗？



发自我的iPhone


------------------ Original ------------------
From: klxdzh ***@***.***&gt;
Date: Mon,Aug 25,2025 10:05 AM
To: zotero-chinese/wiki ***@***.***&gt;
Cc: Ada66688 ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [zotero-chinese/wiki] blog/posts/new-beginning (Discussion #252)





 
那控制面板里面卸载7.0.20会影响软件运行么

**Reply by klxdzh** [reply]
我还没用过这个功能，使用的哪个插件呢，我可以试一下。

**Comment by Ada66688** [comment]
茉莉花，可以网上搜索视频



发自我的iPhone


------------------ Original ------------------
From: klxdzh ***@***.***&gt;
Date: Mon,Aug 25,2025 10:14 AM
To: zotero-chinese/wiki ***@***.***&gt;
Cc: Ada66688 ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [zotero-chinese/wiki] blog/posts/new-beginning (Discussion #252)





 
我还没用过这个功能，使用的哪个插件呢，我可以试一下。

**Reply by klxdzh** [reply]
英文可以抓取，中文不行

**Comment by Ada66688** [comment]
你就只是装了茉莉花嘛，我的都不行，方便问一下你用的什么浏览器吗？



发自我的iPhone


------------------ Original ------------------
From: klxdzh ***@***.***&gt;
Date: Mon,Aug 25,2025 10:49 AM
To: zotero-chinese/wiki ***@***.***&gt;
Cc: Ada66688 ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [zotero-chinese/wiki] blog/posts/new-beginning (Discussion #252)





 
英文可以抓取，中文不行

**Comment by klxdzh** [comment]
我用的edge浏览器，我看了一下，这个好像不是茉莉花实现的，是zotero connector，是zotero官网的Zotero | Your personal research assistant，然后装上就在浏览器上有个插件，点一下插件就保存了




雨送黄昏
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "zotero-chinese/wiki"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2025年8月25日(星期一) 中午11:02
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [zotero-chinese/wiki] blog/posts/new-beginning (Discussion #252)





 你就只是装了茉莉花嘛，我的都不行，方便问一下你用的什么浏览器吗？ 
 
 
 
 发自我的iPhone 
 
 
 ------------------ Original ------------------ 
 From: klxdzh ***@***.***&amp;gt; 
 Date: Mon,Aug 25,2025 10:49 AM 
 To: zotero-chinese/wiki ***@***.***&amp;gt; 
 Cc: Ada66688 ***@***.***&amp;gt;, Comment ***@***.***&amp;gt; 
 Subject: Re: [zotero-chinese/wiki] blog/posts/new-beginning (Discussion #252) 
 
 
 
 
 
   
 英文可以抓取，中文不行

**Comment by Ada66688** [comment]
我装了，还是不行，谢谢啦！



发自我的iPhone


------------------ Original ------------------
From: klxdzh ***@***.***&gt;
Date: Mon,Aug 25,2025 11:17 AM
To: zotero-chinese/wiki ***@***.***&gt;
Cc: Ada66688 ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [zotero-chinese/wiki] blog/posts/new-beginning (Discussion #252)





 我用的edge浏览器，我看了一下，这个好像不是茉莉花实现的，是zotero connector，是zotero官网的Zotero | Your personal research assistant，然后装上就在浏览器上有个插件，点一下插件就保存了 
 
 
 
 
 雨送黄昏 
 ***@***.*** 
 
 
 
 &amp;nbsp; 
 
 
 
 
 ------------------&amp;nbsp;原始邮件&amp;nbsp;------------------ 
 发件人:                                                                                                                        "zotero-chinese/wiki"                                                                                    ***@***.***&amp;gt;; 
 发送时间:&amp;nbsp;2025年8月25日(星期一) 中午11:02 
 ***@***.***&amp;gt;; 
 ***@***.******@***.***&amp;gt;; 
 主题:&amp;nbsp;Re: [zotero-chinese/wiki] blog/posts/new-beginning (Discussion #252) 
 
 
 
 
 
  你就只是装了茉莉花嘛，我的都不行，方便问一下你用的什么浏览器吗？  
   
   
   
  发自我的iPhone  
   
   
  ------------------ Original ------------------  
  From: klxdzh ***@***.***&amp;amp;gt;  
  Date: Mon,Aug 25,2025 10:49 AM  
  To: zotero-chinese/wiki ***@***.***&amp;amp;gt;  
  Cc: Ada66688 ***@***.***&amp;amp;gt;, Comment ***@***.***&amp;amp;gt;  
  Subject: Re: [zotero-chinese/wiki] blog/posts/new-beginning (Discussion #252)  
   
   
   
   
   
     
  英文可以抓取，中文不行
